runners/flink/1.15/src/test/java/org/apache/beam/runners/flink/transform/sql/VerifyingTableSinkFactory.java

```diff
@@ -231,7 +238,7 @@ private static ExpectedResultTable readRowsFromFile(String path, boolean hasHead
     }
   }
 
-  interface SerializableDynamicTableSink extends DynamicTableSink, Serializable {
+  interface SerializableDynamicTableSink extends DynamicTableSink, SupportsOverwrite, Serializable {
```

Comment:
The purpose of the test is to make sure we allow all the statements starting with `INSERT`. So as long as both `INSERT INTO` and `INSERT OVERWRITE` got passed to the TableEnv, the logic we want to test is verified. Therefore, we don't need to cover all the possible syntaxes of `INSERT` because `SqlPTransform` does not deal with them except pass them through.